Formula/imageoptim-cli.rb

```diff
@@ -0,0 +1,23 @@
+require "language/node"
+
+class ImageoptimCli < Formula
+  desc "CLI for ImageOptim, ImageAlpha and JPEGmini"
+  homepage "https://jamiemason.github.io/ImageOptim-CLI/"
+  url "https://github.com/JamieMason/ImageOptim-CLI/archive/2.0.2.tar.gz"
+  sha256 "65ccf26cf45012e20877bf7985b1f9d99cfef4ac3d0f1209dd6fa04ef02056ff"
+
+  depends_on "node" => :build
+
+  def install
+    inreplace "package.json", /("build:bin": "nexe .*)(")/, "\\1 --enableStdIn ''\\2"
+
+    system "npm", "install", *Language::Node.local_npm_install_args
```

Comment:
You could use [`npm ci`](https://docs.npmjs.com/cli/ci#description) to get faster `package-lock.json` based installs here.

```ruby
system "npm", "ci", *Language::Node.local_npm_install_args
```